src/rust/engine/src/nodes.rs

```diff
@@ -1436,6 +1436,7 @@ pub enum NodeOutput {
   DirectoryListing(Arc<DirectoryListing>),
   LinkDest(LinkDest),
   ProcessResult(Box<ProcessResult>),
+  #[allow(clippy::rc_buffer)]
```

Comment:
If this needs another run through CI, referring to https://github.com/rust-lang/rust-clippy/issues/6170 here might be good.

Reply:
Will add. I will merge after business hours in any event.

Reply:
done